docs/index.md

```diff
@@ -1,36 +1,62 @@
 ---
 layout: ""
-page_title: "Provider: kentik-cloudexport"
+page_title: "Provider: Kentik CloudExport"
```

Comment:
The actual name is kentik-cloudexport

Reply:
I had the same comment, however I've looked at documentation of other Terraform providers (e.g. [AWS](https://registry.terraform.io/modules/terraform-aws-modules/vpc/aws/latest)) and this seems to be a common style. We should go with the flow here :). "Cloud Export" though should always be  2 words and no CamelCase (in any text meant for human consumption ... as opposed to code).

Reply:
Kept display name just like in other TF providers